app/activeprojectmanager.cpp

```diff
@@ -108,9 +125,9 @@ bool ActiveProjectManager::forceLoad( const QString &filePath, bool force )
     res = mQgsProject->read( filePath );
     mActiveLayer.resetActiveLayer();
     mMapThemeModel.reloadMapThemes( mQgsProject );
-    mProject = mLocalProjectsManager.projectFromProjectFilePath( filePath );
+    mProject = mLocalProjectsManager.projectPtrFromProjectFilePath( filePath );
```

Comment:
I would keep original name of function "Ptr" is something you will see in your code-editor and doesn't bring any value to the reader of the code

Reply:
Yeah, but there are actually two functions, one (old one) is `projectFromProjectFilePath` which returns a copy and this new one `projectPtrFromProjectFilePath` that returns pointer to original data. So I could not keep the same name

Reply:
Addressed in refactoring based on review